src/imitation/rewards/reward_nets.py

```diff
@@ -168,50 +71,63 @@ def _eval_reward(
             action=action,
             next_state=next_state,
             done=done,
-            device=self.device(),
-            scale=self.scale,
+            device=self.device,
+            scale=self.normalize_images,
```

Comment:
Consider renaming `scale` in `disc_rew_preprocess_inputs`?

Reply:
Done, and also renamed it in the discriminator nets to make everything consistent